doc/errors.rst

```diff
@@ -47,13 +47,30 @@ One case that is not easily covered by the assert statement is asserting Excepti
 
 	  assert caught.exception.param == 'some_value'
 
+:func:`slash.assert_raises` will raise ``TestFailed`` exception in case the expected exception was not raised:
+
+.. code:: python
+
+	 >>> with slash.assert_raises(Exception) as caught: # doctest: +IGNORE_EXCEPTION_DETAIL
+	 ...    pass
+	 Traceback (most recent call last):
+	     ...
+	 TestFailed: ...
+
+In case the test should allow specific exception, but not enfore it (ex. timing issue), you can use :func:`slash.allow_raises`:
```

Comment:
```
In a case where the test author wants to allow a specific exception but not to enforce its propagation (e.g. allowing a timing issue to be present), :func:`slash.allow_raises` can be used.
```
Also add a note and a link explaining the relation of this to handling_exceptions